spec/string-indices-spec.js

```diff
@@ -0,0 +1,17 @@
+const chai = require( 'chai' )
+const { expect } = require( 'chai' )
+const {stringIndices} = require( '../src/string-indices' )
+
+describe( 'stringIndices', () => {
+  'use strict'
+
+  it( 'exists', () => {
+    expect( stringIndices ).to.be.a( 'function' )
+  })
+  it('returns an empty string if the supplied index is longer or shorter than the length of the string.', () => {
```

Comment:
0 is not a negative number and putting in 0 should also return an empty string... I will reconsider how this test is written though.